react/src/components/Products.jsx

```diff
@@ -7,6 +7,7 @@ import measureRequestDuration from '../utils/measureRequestDuration';
 import Loader from 'react-loader-spinner';
 import ProductCard from './ProductCard';
 import { useState, useEffect } from 'react';
+import { LuWheatOff, LuSalad } from "react-icons/lu";
```

Comment:
same

Reply:
I tried this to see if the bundle size would increase

Reply:
ah I see. So this PR itself will be used in demo?
Shouldn't we then expect some kind of codecov bot comment here about bundle size?

Reply:
I'm testing this locally as well:
 npm test -- --coverage --reporters=jest-junit 

Thanks for tips about debugging, I'll try those as well